shell/apps/remote-planning/runtime/mock-slot-composer.js

```diff
@@ -9,8 +9,11 @@
  */
 'use strict';
 
-import {SlotComposer} from '../../../../runtime/ts-build/slot-composer.js';
-import {SlotDomConsumer} from '../../../../runtime/ts-build/slot-dom-consumer.js';
+const arcs = '../../../..';
+const runtime = '${arcs}/build/runtime';
+
+import {SlotComposer} from '${runtime}/slot-composer.js';
+import {SlotDomConsumer} from '${runtime}/slot-dom-consumer.js';
```

Comment:
The 2 imports and the const runtime above need to be back-ticked.

Reply:
Travis found this. Fixed.